### PR TITLE
PMM-9473 Extend MongoDB checks.

### PR DESCRIPTION
**Pull request by PavelKhripkov**
PMM-9473

Build: SUBMODULES-2347

- [ ] https://github.com/percona-platform/saas/pull/60
- [ ] https://github.com/percona-platform/platform/pull/386
- [ ] https://github.com/percona/pmm/pull/826
- [ ] https://github.com/percona/pmm-managed/pull/998

### CODE REVIEW

File: actions/mongodb_query_admincommand_action_test.go [hunk 1]
```diff
@@ -18,238 +18,296 @@ package actions
...
+func convertToObjxMap(t *testing.T, b []byte) objx.Map {
+	t.Helper()
+	data, err := agentpb.UnmarshalActionQueryResult(b)
+	require.NoError(t, err)
+	t.Log(spew.Sdump(data))
+	assert.Len(t, data, 1)
+	return data[0]
+}
+
+func getParameterAssertions(t *testing.T, b []byte) { //nolint:thelper
```
Comment (artemgavrilov) [inline actions/mongodb_query_admincommand_action_test.go:199]: Why you don't want to mark these functions as helpers?

**Review by artemgavrilov — APPROVED** [review]

File: actions/mongodb_query_admincommand_action_test.go (hunk 1 above)
Comment (PavelKhripkov) [reply actions/mongodb_query_admincommand_action_test.go:199]: I'm not sure are they helpers or tests themself. From docs for helpers: _When printing file and line information, that function will be skipped._ So I decided there is no need to mark them as helpers.

Comment (artemgavrilov) [reply actions/mongodb_query_admincommand_action_test.go:199]: Yep, it seems that you're right.

**Review by idoqo — APPROVED** [review]

File: actions/mongodb_query_admincommand_action_test.go [hunk 2]
```diff
@@ -18,238 +18,296 @@ package actions
...
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
+	"go.mongodb.org/mongo-driver/mongo"
 
-	"github.com/percona/pmm-agent/utils/templates"
 	"github.com/percona/pmm-agent/utils/tests"
 )
 
-func TestMongoDBBuildinfo(t *testing.T) {
+func TestMongoDBActions(t *testing.T) {
```
Comment (BupycHuk) [inline actions/mongodb_query_admincommand_action_test.go:36]: looks like we could get rid of these duplicates just by using loop.

**Review by BupycHuk — APPROVED** [review]

File: actions/mongodb_query_admincommand_action_test.go (hunk 2 above)
Comment (BupycHuk) [reply actions/mongodb_query_admincommand_action_test.go:36]: as far as I see they are same, what is different?
I meant using loop for array with all DSNs.

Comment (PavelKhripkov) [reply actions/mongodb_query_admincommand_action_test.go:36]: With one DSN used `getCmdLineOptsAssertionsWithAuth`, with another `getCmdLineOptsAssertionsWithSSL` and with another one `getCmdLineOptsAssertionsWithoutAuth`

Comment (BupycHuk) [reply actions/mongodb_query_admincommand_action_test.go:36]: ok

**Pull request merged**